### PR TITLE
Add blank Libero model

### PR DESCRIPTION
**Pull request by diversemix**
The plugin to Archi appears to use the top level /model and /images directories.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
I downloaded https://www.archimatetool.com/download/ but it doesn't find any `*.archimate` files to open. Missing plugins?

Also, could build a pipeline to render the diagrams on commit, but not convinced this enables collaboration on the maintenance of the diagrams. Any equivalent web-based tool?

**Comment by BlueReZZ** [comment]
Options we know about are Google Drawing (good for collaboration, bad for standardised blocks etc.) and https://structurizr.com/ which the C4 people use. A web-based editor for archimate would be the best solution

**Comment by diversemix** [comment]
> I downloaded https://www.archimatetool.com/download/ but it doesn't find any `*.archimate` files to open. Missing plugins?
> 
> Also, could build a pipeline to render the diagrams on commit, but not convinced this enables collaboration on the maintenance of the diagrams. Any equivalent web-based tool?

Yes ... let me add this to a README

File: model/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Libero Model
+
+## Archimate 
+
+> Archi® is a free, open source, cross-platform tool and editor to create ArchiMate models.
```
Comment (giorgiosironi) [inline model/README.md:5]: Discovered this is a [separate language from UML](https://en.wikipedia.org/wiki/ArchiMate):

> UML and BPMN are meant for a specific use and they are quite heavy – containing about 150 (UML) and 250 (BPMN) modeling concepts whereas ArchiMate works with just about 50 (in version 2.0). The goal of ArchiMate is to be ”as small as possible”, not to cover every edge scenario imaginable.

**Comment by giorgiosironi** [comment]
After installing the plugin, I got up to:
Screenshot from 2019-09-11 09-47-51
I tried opening the various `.xml` files with the `File -> Open...` menu but the content of the windows doesn't change. The top `folder.xml` in particular is the only one that gives an error:
Screenshot from 2019-09-11 09-50-46

**Comment by giorgiosironi** [comment]
> A web-based editor for archimate would be the best solution

I tried https://www.simplyarchimate.com/ which is $12/month:

Screenshot_2019-09-13 SimplyArchimate® - Simple and Intuitive modelling and mindmapping

**Comment by diversemix** [comment]
@giorgiosironi ... agreed anything web based will be inherently a lot more collaborative than a file based solution. I can't figure out from the site tho, if you can have multiple users editing a model at once?

**Comment by BlueReZZ** [comment]
> which is $12/month:

happy to pay that if it makes life easier

**Comment by diversemix** [comment]
@giorgiosironi  ... I see no way of viewing the model ... it looks as though its just a drawing tool - which if true is sad :cry:

**Pull request closed without merging**